setup.cfg

```diff
@@ -1,3 +1,7 @@
 [flake8]
 max-line-length=160
 exclude=migrations,docs,downloads
+
+# H233: Python 3.x incompatible use of print operator
```

Comment:
I think we can remove H233 (It's not triggered anywhere when I run flake8). I'd prefer to remove E251 to keep consistent with all of our other projects, but up to you if you feel strongly.